content/_index.md

```diff
@@ -1,64 +1,56 @@
 ---
 type: index
-title: IPFS Documentation
+title: libp2p Documentation
 ---
 
-Welcome to the IPFS documentation portal! Whether you’re just learning about IPFS or are looking for detailed reference information, this is the place to start. You might have noticed that IPFS is a project with a big scope — and a *lot* of different tools, sites, and code.
+Welcome to the libp2p documentation portal! Whether you’re just learning how to build peer-to-peer systems with libp2p, want to dive into peer-to-peer concepts and solutions, or are looking for detailed reference information, this is the place to start.
 
 ## Introduction
 
-Head over to the [introduction](/introduction) section for an introduction to the basics of IPFS and a guide to getting up and running.
+Head over to the [introduction](/introduction) section for an introduction to the basics of libp2p, an overview of the problems it addresses, and a guide to getting up and running.
+
 
 
 ## Guides
 
-The guides section has an overview of major concepts in IPFS, guides, and example projects demonstrating various ways to use IPFS.
+The guides section has an overview of major concepts in libp2p, guides, and example projects demonstrating various ways to use libp2p to build applications.
 
-In particular, IPFS is a complex system that aims to change a lot of things about how we use the internet, so it naturally comes with a lot of new concepts. Check the [concepts](/guides/concepts) section to learn more about the major architectural pieces of IPFS and about terms and ideas associated with distributed filesystes.
+In particular, check the [concepts](/guides/concepts) section to learn more about the major architectural pieces of libp2p and about terms and ideas associated with peer-to-peer systems in general, as well as those specific to libp2p.
```

Comment:
This links to an auto-generated blank page.  See my previous comment about "Coming soon..."  Not critical for merging this, just a suggestion.